app/views/propertyrepresentation/revokeAgentSummary.scala.html

```diff
@@ -18,28 +18,25 @@
 @import binders.pagination._
 @import binders.propertylinks._
 @import config.ApplicationConfig
+
+@this(mainLayout: views.html.mainLayout)
+
 @(
  action: AgentRevokeBulkAction,
  agentOrganisation: String
 )(implicit request: AuthenticatedRequest[_], messages: Messages, config: ApplicationConfig)
 
 @heading = @{ Messages("propertyRepresentation.agentRemoveSummary.heading") }
 
-@main_template(
-heading,
-contentHeaderText = Some(heading),
-contentHeaderHidden = true
+@mainLayout(
+ heading,
```

Comment:
here it would be good to explicitly specify the name of the parameter

`pageTitle = heading`